### PR TITLE
Fixed code language pt-BR

### PR DESCRIPTION
**Pull request by allancmello**
Follow the pull request, with the correction of the Brazilian Portuguese language code according to the i18next documentation. Corrected to "pt-BR" as per documentation.

My apologies for the previous pull request.

### CODE REVIEW

File: src/utils/i18n.ts [hunk 1]
```diff
@@ -24,6 +24,22 @@ i18next.use(I18nextBrowserLanguageDetector).init({
         'Postal Code': '郵便番号',
       },
     },
+    pt-BR: {
```
Comment (hideokamoto) [inline src/utils/i18n.ts:27]: I think we need to replace this code from `pt-BR` to `'pt-BR'` to resolve this error:
https://github.com/wpkyoto/stripe-pwa-elements/actions/runs/5132459150/jobs/9255449130#step:5:35

**Review by hideokamoto — CHANGES_REQUESTED** [review]
Thanks for making this Pull Request!
To pass our Unit Test on the GitHub Actions, we need to update one line of code.
Would you check my review comment and update your Pull Request?

SIncerely.

File: src/utils/i18n.ts (hunk 1 above)
Comment (allancmello) [reply src/utils/i18n.ts]: Hi Kamoto,

Pull request made according to your guidelines.

Thanks

**Comment by hideokamoto** [comment]
@allancmello 
Thanks!

**Pull request merged**